docs/generator.go

```diff
@@ -22,23 +23,19 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	// detect if file exists and create file if not exists
-	if _, err := os.Stat(tmpl); os.IsNotExist(err) {
-		file, err := os.Create(tmpl)
-		if err != nil {
-			log.Fatal(err)
-		}
-		file.Close()
+
+	f, err := os.OpenFile(tmpl, os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
That removes the os.O_APPEND property. Is that correct?

Reply:
Previously that was happening to me but now when I do it the whole docs are getting generated that previous problem might be due to some other issue. So, it works fine.

Reply:
Sounds good.